content/en/docs/Core concepts/sharding.md

```diff
@@ -1,23 +1,81 @@
 ---
 title: "Sharding"
-date: 2020-11-02
-weight: 500
+date: 2020-11-03
+weight: 400
+description: >
+  Output sharding
 ---
 
-# TBD - these are just notes
+### Overview
 
-The `[output.sharding]` configuration in the TOML file tells which field of the records must be
-used to calculate the sharding.
+Baker supports partitioning the records it processes into smaller subsets each
+of which is forwarded to an output shard: _divide and rule_.
 
-The field name, transformed to `FieldIndex` thanks to the `FieldByName` function (see
-[below](#fieldbyname)), is the index of the map that corresponds to a sharding function.
+When sharding is enabled, the shards, which are just multiple instances of the
+same output component, run concurrently. Each of them only get to process a 
+specific subset of of records, based on a the value a specific field has. This
+horizontal partioning allows to get the most of the resources at your disposal,
+since you can perform more work at the same time.
 
-The sharding function receives a Record and returns a sharding value (`uint64`) that Baker
-uses to send the Record to an output process.
+#### Limitations
```

Comment:
I'd maybe also add that sharding works on a single field

Reply:
Not sure it's a limitation though. the overview clearly says sharding works with a single field.
But ok for safety i've added:
> Only a single field can be used for sharding